spec/features/pair_spec.rb

```diff
@@ -21,8 +21,14 @@
       expect(client.instance_variable_get(:@tokens)).to be_empty
     end
     it "should have a pos token after pairing" do
-      client.pair_pos_client(claimCode)  
-      expect(client.instance_variable_get(:@tokens)['pos']).not_to be_empty
+      sleep 1 # rate limit compliance
```

Comment:
We really shouldn't have to do this if we clear the claim codes between runs. So far I have only occasionally run into rate limiter problems with a single run.